src/lib.rs

```diff
@@ -613,6 +613,44 @@ impl Client {
             )))))
         }
     }
+
+    /// Get data and update the expiration time of an existing data by key
+    ///
+    /// If the key is found, `Some(Value)` is returned, describing the metadata and data of the key.
+    ///
+    /// Otherwise, [`Error`] is returned.
+    pub async fn gat<K>(&mut self, exptime: i32, key: K) -> Result<Option<Value>, Error>
```

Comment:
How about putting `key: K` after `&mut self` as the second parameter? Just want to make it be aligned with other APIs.

Reply:
Hello, I see such an API for memcached. Let's look at the convenience of your offer. Your preference also applies. You can change it if there is such a need. 

For example, I am more inclined to look at the memcached documentation, but convenience may be more important.
[gat](https://github.com/memcached/memcached/blob/5609673ed29db98a377749fab469fe80777de8fd/doc/protocol.txt#L431)

Reply:
@drinkbeer ,could you take a look at the signature? You probably meant this API. Thanks for pointing this out.